### ISSUE TITLE
Image zoom doesn't work

### ISSUE THREAD

**Issue by doochik** [ldn0x7dc/react-native-gallery#6]
The example

```
class Foo extends React.Component {
    render() {
        return (
            <Gallery
                images={[
                    'http://img.verbapp.me/s3/upload/photos/3/92bcf9ec25366d4e4bcfe81348c6cb56708e0484.jpg'
                ]}
            />
        );
   }
}
```

Pinch zoom doens't work. The image will zoom on double tap only.
Is this bug or I should setup some props?

**Comment by ldn0x7dc** [comment]
@doochik 
Should be a bug.
Could you post more info here like react-native version, ios/android version?
If you are using an iOS simulator, a pinch must be triggered with _alt_ pressed.

**Comment by doochik** [comment]
rn 0.29, react-native-gallery 0.0.15
try on iPhone SE iOS 9.3.2 and iOS Emulator 5s iOS 9.3.2

**Comment by chadwilken** [comment]
Having the same issue.

**Comment by ldn0x7dc** [comment]
@doochik @chadwilken 
If your issue is caused by a Modal being parent view, it should be fixed by latest release(0.0.17). Also, the iOS simulator is able to pinch with `alt` pressed.

**Comment by chadwilken** [comment]
@ldn0x7dc feel free to close, upgrading to 0.0.17 fixed the issue. Thanks for your work, love the library.
